.github/workflows/no-response.yml

```diff
@@ -1,34 +1,35 @@
 # A workflow to close issues where the author hasn't responded to a request for
-# more information; see https://github.com/godofredoc/no-response for docs.
+# more information; see https://github.com/actions/stale.
 
 name: No Response
 
-# Both `issue_comment` and `scheduled` event types are required.
+# Run as a daily cron.
 on:
-  issue_comment:
-    types: [created]
   schedule:
-    # Schedule for every day at 8am
+    # Every day at 8am
     - cron: '0 8 * * *'
 
 # All permissions not specified are set to 'none'.
 permissions:
   issues: write
+  pull-requests: write
 
 jobs:
-  noResponse:
+  no-response:
     runs-on: ubuntu-latest
     if: ${{ github.repository_owner == 'dart-lang' }}
     steps:
-      - uses: godofredoc/no-response@0ce2dc0e63e1c7d2b87752ceed091f6d32c9df09
+      - uses: actions/stale@1160a2240286f5da8ec72b1c0816ce2481aabf84
         with:
-          responseRequiredLabel: "needs-info"
-          responseRequiredColor: 4774bc
-          daysUntilClose: 14
-          # Comment to post when closing an Issue for lack of response.
-          closeComment: >
-            Without additional information we're not able to resolve this
-            issue, so it will be closed at this time. You're still free to add
-            more info and respond to any questions above, though. We'll reopen
-            the case if you do. Thanks for your contribution!
-          token: ${{ github.token }}
+          days-before-stale: -1
```

Comment:
I'm curious why `days-before-stale` is set to -1. Worth a doc comment?

Reply:
Sure - I'll update our template for this bot config. By default this bot wants to mark issues + PRs w/o activity as 'stale' after 60 days, and w/o any other updates close them 7 days later. Here we're disabling the auto-stale behavior.

Reply:
https://github.com/dart-lang/ecosystem/pull/102/commits/79264563715b047ca86c0edd9403128d78d1ca06